### PR TITLE
feat(material-experimental/theming): provide hook for formatting toke…

### PR DESCRIPTION
**Pull request by wagnermaciel**
…ns internally

### CODE REVIEW

File: src/material-experimental/theming/_format-tokens.scss [hunk 1]
```diff
@@ -0,0 +1,5 @@
+// DO NOT REMOVE
+// This function is used internally.
+@function format($systems) {
```
Comment (crisbeto) [inline src/material-experimental/theming/_format-tokens.scss:3]: Should we change the name to indicate that this is a private API?

Comment (wagnermaciel) [reply src/material-experimental/theming/_format-tokens.scss]: Done. Let me know if you would like a different name

File: src/material-experimental/theming/_format-tokens.scss [hunk 2]
```diff
@@ -0,0 +1,5 @@
+// DO NOT REMOVE
+// This function is used internally.
+@function private-internal-format($systems) {
```
Comment (crisbeto) [inline src/material-experimental/theming/_format-tokens.scss:3]: Maybe something like `private-format-tokens`? `format` is a bit generic on its own.

**Review by crisbeto — APPROVED** [review]
LGTM

**Pull request merged**